ModuleFast.tests.ps1

```diff
@@ -278,6 +278,22 @@ Describe 'Get-ModuleFastPlan' -Tag 'E2E' {
             $actual.ModuleVersion | Should -Be '1.99.0' #Nuget changes this to 1.99
           }
         },
+        # Special cases where the upper bound is specified as exclusive, ignore prereleases
+        @{
+          Spec       = 'PrereleaseTest!<0.0.2'
+          ModuleName = 'PrereleaseTest'
+          Check      = {
+            $actual.ModuleVersion | Should -Be '0.0.1' #Install module version is 0.4.15.0 but nuget truncates this
```

Comment:
The comment on this line is a lie ;-)

Reply:
May also need a test with strictSemver on

Reply:
Fixed and test added c03faeece6a5b9341bd090f9def093e9c3a59f65